plugins/backstage-plugin-flux/src/hooks/useToggleSuspendResource.ts

```diff
@@ -124,9 +135,16 @@ export function useToggleSuspendResource(
 ) {
   const kubernetesApi = useApi(kubernetesApiRef);
   const alertApi = useApi(alertApiRef);
+  const { data } = useGetUserInfo();
+  const user =
+    data?.profile?.email ||
+    data?.profile?.displayName ||
+    data?.userId ||
+    'guest user';
 
   const [{ loading }, toggleSuspend] = useAsyncFn(
-    () => toggleSuspendResource(resource, kubernetesApi, alertApi, suspend),
+    () =>
+      toggleSuspendResource(resource, kubernetesApi, alertApi, suspend, user),
     [resource, kubernetesApi, alert],
```

Comment:
We might need to add `user` to the dep list, I noticed I still get "guest" sometimes. 😕 

I'm not sure why eslint is not detecting this dep list.